index.html

```diff
@@ -0,0 +1,309 @@
+                    Technologies, where I delved into HTML, CSS, JavaScript, and the React framework. It was
+                    exhilarating to see my skills evolve and solidify. This experience marked my first real training
+                    in the field, and I enjoyed it so much that I resolved to pursue a career in front-end
+                    development.
+                </p>
+
+                <p>Realizing the need for further training to delve into intricacies, best practices, and advanced
+                    topics in front-end development to enhance my readiness for the competitive job market, I
+                    enrolled in Foothill Technology Solutions' training program. Currently, I am expanding my skills
+                    and gaining more hands-on experience in the field.</p>
+            </article>
+
+            <article class="beliefs-article">
+                <p>To me, front-end development is more than just coding; it's an ever-evolving journey that demands
+                    creativity, adaptability, and a deep understanding of browser dynamics. I'm driven by the thrill
+                    of mastering this complex craft and shaping meaningful digital experiences.
+                </p>
+                <p>Looking towards the future fills me with excitement and anticipation. Each day brings new
+                    opportunities for learning, growth, and innovation. I am eager to embrace whatever surprises lie
+                    ahead, prepared to leave my mark in the ever-evolving tech universe.</p>
+            </article>
+        </section>
+
+
+        <!-- skills section -->
+        <section class="skills-page" id="skills-page">
+            <h2>Skills</h2>
+            <div class="skills-content">
+                <img src="assets/images/backgrounds/hello-world.gif">
+                <ul class="skills">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>JavaScript</li>
+                    <li>ECMAScript</li>
+                    <li>React js</li>
+                    <li>Java</li>
+                    <li>C</li>
+                    <li>Python</li>
+                    <li>SQL</li>
+                    <li>MySQL</li>
+                    <li>Data Structures</li>
+                    <li>Databasees</li>
+                    <li>GIT</li>
+                    <li>Github</li>
+                    <li>Linux</li>
+                    <li>Cypress</li>
+                    <li>Firebase</li>
+                    <li>Android Studio</li>
+                    <li>Agile development</li>
+                </ul>
+            </div>
+        </section>
+
+
+        <!-- projects section -->
+        <section class="projects-page" id="projects-page">
+            <h2> My Projects</h2>
+            <ul>
+                <li>
+                    <div class="img-container">
+                        <img src="assets/images/projects/secret-code-game.png">
```

Comment:
img should have **alt** attribute

Reply:
Oh yeah, I forgot that! Thanks, I'll add it